app/modules/product/card/variants/bigger/BiggerCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react'
+import cl from './BiggerCard.module.scss'
+import Image from 'next/image'
+import names from 'classnames'
+import { lato } from '@/app/styles/fonts'
+import { Product } from '@/app/modules/product/types/product.type'
+import Link from 'next/link'
+import AddToCartButton from '@/app/modules/cart/add-to-cart-button/AddToCartButton'
+
```

Comment:
Better not push empty files to the git, I'm talking about `BiggerCard.module.scss`